RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -0,0 +1,426 @@
+                     (startHeight == 0 ? 0 : GetSysFeeAmountForHeight(snapshot.Blocks, startHeight - 1)));
+        }
+
+        private bool AddClaims(JArray claimableOutput, ref Fixed8 runningTotal, int maxClaims,
+            Snapshot snapshot, DataCache<UInt256, SpentCoinState> storeSpentCoins,
+            KeyValuePair<UserSystemAssetCoinOutputsKey, UserSystemAssetCoinOutputs> claimableInTx)
+        {
+            foreach (var claimTransaction in claimableInTx.Value.AmountByTxIndex)
+            {
+                var utxo = new JObject();
+                var txId = claimableInTx.Key.TxHash.ToString().Substring(2);
+                utxo["txid"] = txId;
+                utxo["n"] = claimTransaction.Key;
+                var spentCoinState = storeSpentCoins.TryGet(claimableInTx.Key.TxHash);
+                var startHeight = spentCoinState.TransactionHeight;
+                var endHeight = spentCoinState.Items[claimTransaction.Key];
+                CalculateClaimable(snapshot, claimTransaction.Value, startHeight, endHeight, out var generated,
+                    out var sysFee);
+                var unclaimed = generated + sysFee;
+                utxo["value"] = (double) (decimal) claimTransaction.Value;
+                utxo["start_height"] = startHeight;
+                utxo["end_height"] = endHeight;
+                utxo["generated"] = (double) (decimal) generated;
+                utxo["sys_fee"] = (double) (decimal) sysFee;
+                utxo["unclaimed"] = (double) (decimal) unclaimed;
+                runningTotal += unclaimed;
+                claimableOutput.Add(utxo);
+                if (claimableOutput.Count > maxClaims)
+                    return false;
+            }
+
+            return true;
+        }
+
+        private JObject ProcessGetClaimableSpents(JArray parameters)
+        {
+            UInt160 scriptHash = GetScriptHashFromParam(parameters[0].AsString());
+            var dbCache = new DbCache<UserSystemAssetCoinOutputsKey, UserSystemAssetCoinOutputs>(
+                _db, null, null, SystemAssetSpentUnclaimedCoinsPrefix);
+
+            JObject json = new JObject();
+            JArray claimable = new JArray();
+            json["claimable"] = claimable;
+            json["address"] = scriptHash.ToAddress();
+
+            Fixed8 totalUnclaimed = Fixed8.Zero;
+            using (Snapshot snapshot = Blockchain.Singleton.GetSnapshot())
+            {
+                var storeSpentCoins = snapshot.SpentCoins;
+                byte[] prefix = new [] { (byte) 1 }.Concat(scriptHash.ToArray()).ToArray();
+                foreach (var claimableInTx in dbCache.Find(prefix))
+                    if (!AddClaims(claimable, ref totalUnclaimed, _rpcMaxUnspents, snapshot, storeSpentCoins,
+                        claimableInTx))
+                        break;
+            }
+            json["unclaimed"] = (double) (decimal) totalUnclaimed;
```

Comment:
@jsolman, maybe this should be `json["available"]` in order to be consistent with `ProcessGetUnclaimed()`

Reply:
It currently is matching the neoscan api to use `unvlaimed`. I think it is better to match their response to make less work for those that already have code that uses their responses.

Reply:
I think that it is good that if matches.
You are right, Jeff.

In addition, I think that there is not a standard, I was just checking the NEL explorer and they have different types of calls: http://www.xiaoyaoji.cn/share/1H0gjTDtfk/1dLQ8CDVdd